stubs/redis/redis/lock.pyi

```diff
@@ -13,7 +13,7 @@ class Lock:
         timeout: None | int | float = ...,
         sleep: float = ...,
         blocking: bool = ...,
-        blocking_timeout: bool | None = ...,
+        blocking_timeout: None | int | float = ...,
```

Comment:
Nit:

```suggestion
        blocking_timeout: float | None = ...,
```

We put `None` last in typeshed and `int` is redundant in argument types as it's a sub-class of `float` for type-checking purposes.

Reply:
I just followed the style in the same module but then would it better to fix other similar places in the module?